changelog/2748.bugfix.rst

```diff
@@ -0,0 +1 @@
+Do not attempt to save a FITS header comment for a keyword which is not in the header. This prevents an error on saving some maps after the metadata had been modified but not the comments.
```

Comment:
Here it should be "for a keyword **that** is not in the header".

Reply:
I was about to disagree, but it [appears you're right](https://en.oxforddictionaries.com/usage/that-or-which). Note though:

> in British English, the word *which* is often used interchangeably with the restrictive *that* ... [this] is not strictly incorrect in American English, but it is generally avoided, especially in formal writing